modules/install-upgrade/pages/upgrade-guide.adoc

```diff
@@ -0,0 +1,554 @@
+. To fetch the updated images, run the following command:
++
+[source,bash]
+----
+docker-compose pull
+----
++
+. To ensure a clean upgrade, stop and remove your existing Pulsar containers. Run:
++
+[source,bash]
+----
+docker-compose down
+----
++
+. To start the updated Pulsar deployment, run the following command:
++
+[source,bash]
+----
+docker-compose up -d
+----
++
+This command creates new containers based on the updated image and configurations.
++
+. Verify the upgrade. Check the logs to ensure that Pulsar starts without errors:
++
+[source,bash]
+----
+docker-compose logs -f pulsar
+----
++
+. To confirm that everything is functioning as expected, verify that you can access the Pulsar Admin UI at `http://localhost:8080` (or whichever port you configured).
+. Run health checks with Pulsar's built-in health check commands or APIs to ensure all services are operational.
+
+// known issues?
+
+=== Kubernetes deployment using KAAP Operator
+
+Deploying Luna Streaming on Kubernetes with KAAP (Kubernetes Autoscaling for Apache Pulsar) Operator is a common method for running Pulsar in a cloud-native environment.
+
+For more information, see the xref:kaap-operator::index.adoc[KAAP documentation].
+
+. Back up your existing Pulsar data and configurations to prevent data loss.
+. To save your current Helm release configuration, run the following command:
++
+[source,bash]
+----
+helm get values > pulsar-backup-values.yaml
+----
++
+. To update the DataStax Pulsar Helm chart repository, run the following command:
++
+[source,bash]
+----
+helm repo update
+----
++
+. Open `helm/kaap-stack/values.yaml` and update the image tag to 3.1.0 (or the specific tag you wish to use).
++
+[source,yaml]
+----
+kaap:
+  enabled: true
+  cluster:
+    name: pulsar
+    create: true
+  spec:
+    global:
+      name: pulsar
+      image:
+        datastax/lunastreaming-all: 3.1_4.5
+----
++
+. Review and modify any other configuration parameters that may have changed between versions, such as resource limits, storage classes, and additional components. To modify other configurations, update `values.yaml` as needed. For example, to modify the broker's namespace shedding and splitting configurations, update the following fields:
++
+[source,yaml]
+----
+kaap:
+  enabled: true
+  cluster:
+    name: pulsar
+    create: true
+  spec:
+    global:
+      name: pulsar
+    broker:
+      replicas: 2
+      config:
+        loadBalancerNamespaceBundleSplitConditionHitCountThreshold: 1
+        loadBalancerSheddingConditionHitCountThreshold: 1e
+----
++
+. To upgrade your existing Pulsar installation, run the following Helm command.
+This command assumes the default `pulsar` namespace. If you are using a different namespace, replace `pulsar` with your namespace.
+The `--wait` flag ensures that Helm waits until all pods are ready before completing the upgrade.
++
+[source,bash]
+----
+helm upgrade pulsar --namespace pulsar --values current-values.yaml --wait
```

Comment:
Please update the `helm upgrade` command. Please refer [here](https://docs.google.com/document/d/1mSbx0ig-QPTRe23gRqlqNKvptX5qwrugCAH-TGOYlsY/edit#heading=h.3zbqqcbvk8ok).